client/state/notices/middleware.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { truncate } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { successNotice, errorNotice } from 'state/notices/actions';
+import { getSitePost } from 'state/posts/selectors';
+import {
+	POST_DELETE_FAILURE,
+	POST_DELETE_SUCCESS,
+	POST_SAVE_SUCCESS
+} from 'state/action-types';
+
+export const OBSERVERS = {
```

Comment:
>WHY ARE YOU SHOUTING?

Fun fact, the name for this style is SCREAMING_SNAKE_CASE.

More to the point, this is a [documented style for constant declarations](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions).

>also I'm kind of curious on your name `observer`.

I'm not opposed to `handler`. I think it popped into my head at least once in the course of implementing this. Given the concerns on interpretation, I'll just rename it.